chart/values.yaml

```diff
@@ -118,13 +118,15 @@ waitForInitContainer:
 securityContext:
   runAsUser: 65534
   runAsNonRoot: true
-dlite:
-  linux_name: dlite-linux
-  accountId: "__GLOBAL_DELEGATE_ACCOUNT_ID__"
-  accountSecret: "2f6b0988b6fb3370073c3d0505baee59"
-  managerEndpoint: "http://harness-manager:9090"
-  pollIntervalMs: "1000"
-  parallelWorkers: "10000"
+config:
+  DLITE_NAME: dlite-linux
+  DLITE_MANAGER_ENDPOINT: "http://harness-manager:9090"
+  DLITE_POLL_INTERVAL_MILLISECS: "1000"
+  DLITE_PARALLEL_WORKERS: "10000"
+  SKIP_PREPARE_SERVER: "true"
+  DRONE_TRACE: "true"
+  DRONE_DISTRIBUTED_ENABLED: "true"
```

Comment:
You can remove DRONE_DISTRIBUTED_ENABLED property, it's not required anymore. You can rebase